### ISSUE TITLE
==blenderpath== pointing to absolute location

### ISSUE THREAD

**Issue by aditiapratama** [fsiddi/flamenco#119]
In worker/conrollers.py line 462

```python
 for cmd, val in enumerate(render_command):
        render_command[cmd] = render_command[cmd].replace(
            "==jobpath==",jobpath)
        render_command[cmd] = render_command[cmd].replace(
            "==outputpath==",outputpath)
        render_command[cmd] = render_command[cmd].replace(
            "==blenderpath==", "/shared/software/blender/blender_farm_latest/blender")
```
This causing issue of render failed in workers

**Comment by aditiapratama** [comment]
When I change the blenderpath above with correct path of blender, I've got the following error
```
Read new prefs: /home/cgmonster/.config/blender/2.73/config/userpref.blend
found bundled python: /media/server/blender/2.73/python
read blend: /tmp/1/1//
Error: Failed to read blend file '/tmp/1/1//', not a blend file

[]
```

**Comment by Eibriel** [comment]
Hi aditia, about the first report is a work in progress  change. You can change the code or reset your local repo to a previous commit

**Comment by Eibriel** [comment]
About the second one, the location of the Storage folder has changed to /static/storage but your config.py file points to an old location, check config.py.example. (dismiss this if you reset the repo)

**Comment by Eibriel** [comment]
Fixed 045057cf49fb35964082e569a5adb2a31d7a7cac, now the Workers can read settings from the Manager, *You need to set some parameters in the database*, as following:

On **Manager** database, on **settings** add:

name | value
---------------|---------------
simple_blender_render | {"commands": {"linux": "/path/to/blender"}}
tiled_blender_render | {"commands": {"linux": "/path/to/blender"}}
tiled_blender_render_simple_mix | {"commands": {"linux": "/path/to/blender"}}

The Blender addon also have a basic UI to add settings, if you prefer.

You need to:
* select the Manager where you want to add the Setting,
* press **Add Manager Setting**, and enter the **Name** and **Value**,
* press **Save Preferences** (you can't change the name once the Setting is saved)

**Comment by Eibriel** [comment]
A small change on the ***value***:
```
{"commands": {"default": {"linux": "/path/to/blender"}}}
```

You can add more than one version of blender, for example:
```
{"commands": {"default": {"linux": "/path/to/blender"}, "multiview": {"linux": "/path/to/multiview/blender"}}}
```
